components/gitpod-cli/BUILD.yaml

```diff
@@ -9,7 +9,20 @@ packages:
       - CGO_ENABLED=0
       - GOOS=linux
     deps:
+      - :version
       - components/supervisor-api/go:lib
       - components/gitpod-protocol/go:lib
+    argdeps:
+      - version
     config:
       packaging: app
+    prep:
+      - ["cp", "_deps/components-gitpod-cli--version/version.txt", "pkg/gitpod/version.txt"]
```

Comment:
I see there are still 2 version.txt files

Reply:
There are indeed 2, but I'm not sure if I can do this without them: the embed package we use to get the version, but with it, we can't go to a parent directory, so that's why I did it this way. It is also how [the local app does it](https://github.com/gitpod-io/gitpod/blob/1601f12b358c90db464b8a26824414e0c153cb6d/components/local-app/main.go#L33-L39).

Reply:
@iQQBot just to confirm there is not way to have it only once and reuse across cli? if so let's merge